panels/config.py

```diff
@@ -0,0 +1,12 @@
+from panels import *
+
+panels_config = parse_config(__file__)
+c.include_plugin_config(panels_config)
```

Comment:
As mentioned in the other PR at https://github.com/magfest/ubersystem/pull/1434 the ``c.include_plugin_config()`` method was extended to also look for ``[enums]`` and ``[dates]`` sections.